packages/migrations/src/actions/2023.02.22T09.27.02.delete-personal-org.sql

```diff
@@ -0,0 +1,13 @@
+-- Find and delete all organizations of type PERSONAL that have no projects
+DELETE FROM public.organizations as o
+WHERE
+  o.type = 'PERSONAL'
+  AND (
+    SELECT COUNT(*) from public.projects as p WHERE p.org_id = o.id
+  ) = 0;
```

Comment:
We need to double check that and check how long it will take

Reply:
I checked and it takes 4s to run this as `SELECT`. Since all of the found records (844 in total) are not references to any other object, i don't expect the `DELETE` to take much longer?

Reply:
couldnt you just instead fo

`SELECT id from public.projects where project.org_id = o.id LIMIT 1`?

If there is one row that should be sufficient, no need to get all

Reply:
for sure!

Reply:
i did:
```
AND NOT EXISTS (
    SELECT id from public.projects as p WHERE p.org_id = o.id LIMIT 1
  );
```

even faster 🚀